Rakefile

```diff
@@ -2,50 +2,50 @@
 # Rakefile for agent-payload
 #
 
-protoc_binary="protoc"
-protoc_version="3.5.1"
+protoc_legacy_version="3.5.1"
```

Comment:
nit: May be good to add a comment similar to your PR description in the code for maintainability but not a blocker.